package.json

```diff
@@ -1,9 +1,165 @@
 {
-  "name": "ca-project",
+  "name": "social-media-app",
   "version": "0.0.1",
-  "description": "project for school",
+  "description": "school project",
   "main": "index.js",
+  "dependencies": {
+    "accepts": "^1.3.8",
```

Comment:
Too many dependencies, you should only have bootstrap in here. This usually happens when you run `npm init` on an existing project. Simply delete all of the items in dependencies and `npm i bootstrap` again.

Reply:
fixed that, but nothing came back when I did ``js npm i boostrap``

Reply:
That's because you installed as a devDependency, you can see it there.

```bash
npm uninstall bootstrap
npm install bootstrap
```

Reply:
its in the dependencies now